lib/mix/tasks/appsignal.install.ex

```diff
@@ -183,15 +183,8 @@ defmodule Mix.Tasks.Appsignal.Install do
 
     case File.open(appsignal_config_file_path(), [:write]) do
       {:ok, file} ->
-        case IO.binwrite(file, appsignal_config_file_contents(config)) do
-          :ok ->
-            IO.puts("Success!")
-
-          {:error, reason} ->
-            IO.puts("Failure! #{inspect(reason)}")
-            exit(:shutdown)
-        end
-
+        IO.binwrite(file, appsignal_config_file_contents(config))
```

Comment:
I think this change will fail for older Elixir versions.

See

- https://github.com/appsignal/appsignal-elixir/pull/951
- https://appsignal.slack.com/archives/CNPP953E2/p1718973152367769

Reply:
It won't fail, it will silently swallow the error -- it needs to at least be `:ok = IO.binwrite(...)`, for compatibility with older Elixir versions.